skops/card/_model_card.py

```diff
@@ -162,6 +164,42 @@ def metadata_from_config(config_path: Union[str, Path]) -> CardData:
     return card_data
 
 
+def _load_model(model: Any) -> Any:
+    """Loads the mddel if provided a file path, if already a model instance return it unmodified.
+
+    Parameters
+    ----------
+    model : pathlib.path, str, or sklearn estimator
+        Path/str or the actual model instance. if a Path or str, loads the model on first call.
+
+    Returns
+    -------
+    model : object
+        Model instance.
+
+    """
+
+    if not isinstance(model, (Path, str)):
+        return model
+
+    model_path = Path(model)
+    if not model_path.exists():
+        raise ValueError("Model file does not exist")
```

Comment:
Could you please add a test case for this potential error? Also, `FileNotFoundError` is a more precise error.